### ISSUE TITLE
New template with setting "use only once" can not be selected when switching template

### ISSUE THREAD

**Issue by AndZyk** [processwire/processwire-issues#693]
### Short description of the issue

When you create a template and check in the "Family" tab under "Can this template be used for new pages?" the radio button "Once", you cannot select the new template while switching the template of a existing page. Creating a new page with the new template works as expected. 

### Expected behavior

When trying to switch the template of a existing page, I would expect to be able to select the new template, because it isn't used by a page yet.

### Actual behavior

You cannot select the new template while switching the template of a existing page. Creating a new page with the new template works as expected.

### Steps to reproduce the issue

1. Create a new template.
2. Check in the "Family" tab under "Can this template be used for new pages?" the radio button "Once".
3. Try to switch the template of a existing page.

### Setup/Environment
<strong>Server Details</strong>

|Software|Version|
|------:|:-------|
|ProcessWire|3.0.112|
|PHP|7.1.19|
|Webserver|Apache|
|MySQL|5.7.21|
<details><summary><strong>Server Settings</strong></summary>

|Parameter|Value|
|------:|:-------|
|allow_url_fopen|1|
|max_execution_time|30 (changeable)|
|max_input_nesting_level|64|
|max_input_time|60|
|max_input_vars|1000|
|memory_limit|128M|
|post_max_size|32M|
|upload_max_filesize|32M|
|xdebug||
|xdebug.max_nesting_level||
|mod_rewrite||
|mod_security||
|GD|bundled (2.1.0 compatible)|
|GIF|1|
|JPG|1|
|PNG|1|
|EXIF Support|1|
|FreeType|1|
|Imagick Extension|1|
</details><details><summary><strong>Module Details</strong></summary>

|Module ClassName|Version|
|------:|:-------|
|FieldtypeColor|1.0.8|
|FieldtypeFieldsetGroup|0.0.4|
|FieldtypeFunctional|0.0.1|
|FieldtypeImageMarker|0.1.1|
|FieldtypePhone|3.0.1|
|FieldtypeRepeaterMatrix|0.0.4|
|InputfieldColor|1.0.8|
|InputfieldFunctional|0.0.1|
|InputfieldImageMarker|0.1.1|
|InputfieldPhone|3.0.1|
|InputfieldRepeaterMatrix|0.0.4|
|ProcessWireUpgrade|0.0.7|
|ProcessWireUpgradeCheck|0.0.7|
|TextformatterAutoLinks|0.0.5|
|TracyDebugger|4.11.23|
</details>

**Comment by Toutouwai** [comment]
Duplicate of #685

**Comment by ryancramerdesign** [comment]
Thanks, this should already be fixed in the commit related to the fix for issue #685, so I'll close this one out.

**Comment by AndZyk** [comment]
Thank you for the fix.
